Content.Shared/Storage/EntitySpawnEntry.cs

```diff
@@ -44,25 +44,20 @@ public struct EntitySpawnEntry : IPopulateDefaultValues
     ///     </example>
     /// </summary>
     [ViewVariables(VVAccess.ReadWrite)]
-    [DataField("orGroup")] public string? GroupId;
+    [DataField("orGroup")] public string? GroupId = null;
```

Comment:
Is this one necessary because of the new behaviour?

Reply:
ye, compiler screams at you otherwise